### ISSUE TITLE
recent

### ISSUE THREAD

**Issue by avbx5566** [frifox/gw2dps#49]
can't use this version , will update?

**Comment by frifox** [comment]
Haven't been playing GW2 for few years now so not expecting to push any updates. Since then hairys been maintaining gw2dps so it's up to him (or whoever steps up for the task) now. If he's too busy then that's how it is, no updates.

**Comment by hairys** [comment]
I've been on a bit of a hiatus. I'll update it eventually if nobody else wants to. I've been picking away at it a little bit at a time, adding offset scans to gw2lib. I have most of 32-bit done. I will try to keep it compliant with Chris Cleary's comment on the [AMA](https://www.reddit.com/r/Guildwars2/comments/5svug8/the_head_of_the_snake_devs_here_ask_us_anything/ddi77u2/) and update it to use ImGui.

**Comment by Hansker** [comment]
Is there an old patch that can be used in this version?

**Comment by hairys** [comment]
Not really. When anet puts out a new update for the game, the offsets may change. The new offsets need to be searched for and updated over at [gw2lib](https://bitbucket.org/rafzi/hacklib_gw2). Nothing in the gw2dps repo really needs to change, but you still need to re-compile it after updating the offsets.
